sys/include/net/gnrc/tcp/config.h

```diff
@@ -53,17 +53,17 @@ extern "C" {
  * @{
  */
 /**
- * @brief Timeout duration for user calls. Default is 2 minutes.
+ * @brief Timeout duration in milliseconds for user calls. Default is 2 minutes.
  */
 #ifndef CONFIG_GNRC_TCP_CONNECTION_TIMEOUT_DURATION
-#define CONFIG_GNRC_TCP_CONNECTION_TIMEOUT_DURATION (120U * US_PER_SEC)
+#define CONFIG_GNRC_TCP_CONNECTION_TIMEOUT_DURATION (120U * MS_PER_SEC)
 #endif
 
 /**
- * @brief Maximum segment lifetime (MSL). Default is 30 seconds.
+ * @brief Maximum segment lifetime (MSL) in milliseconds. Default is 30 seconds.
  */
 #ifndef CONFIG_GNRC_TCP_MSL
-#define CONFIG_GNRC_TCP_MSL (30U * US_PER_SEC)
+#define CONFIG_GNRC_TCP_MSL (30U * MS_PER_SEC)
```

Comment:
Would it make sense to have this in seconds (and call the define appropriately)

Reply:
You are right, MSL is usually given in seconds, however it was tracked by xtimer therefore I just wanted the value expressed in the correct unit. Instead of using seconds and convert the value in code, I'd like to rename the define to CONFIG_GNRC_TCP_MSL_MS. Are you happy with that?

Reply:
Yea that would be less ambiguous.

Reply:
Addressed [here ]( https://github.com/RIOT-OS/RIOT/pull/14500/commits/7d7a83918cbbc65ebaddd5b6cbf29cde09c865d8)